modules/ROOT/pages/bat-install-task.adoc

```diff
@@ -5,10 +5,28 @@ endif::[]
 
 The Blackbox Automated Testing (BAT) CLI wrapper is a command line wrapper that you install to execute API tests.
 
-== Prerequisite
+== Prerequisite for Linux and MacOS Systems
 
 Before installing the BAT CLI, ensure that AdoptOpenJDK's OpenJDK 8 is installed on your system.
 
+== Prerequisites for Windows 10 Systems
+
+. Install Windows Subsystem Linux (WSL)
++
+The instructions are at https://docs.microsoft.com/en-us/windows/wsl/install-win10.
+. Install Java 8, cURL, and unzip installed in the Linux distribution that you installed in your WSL.
```

Comment:
Should we change `... and unzip installed in the Linux... ` for `... and unzip in the Linux ...`?